docs/tutorials/react-native.mdx

```diff
@@ -0,0 +1,708 @@
+
+<Tabs>
+  <TabItem value="react-native" label="React Native">
+
+You need to at least set up camera permissions.
+
+On Android add to your `AndroidManifest.xml`:
+
+```xml title=AndroidManifest.xml
+  <uses-permission android:name="android.permission.CAMERA"/>
+```
+
+For audio you'll need the `RECORD_AUDIO` permission:
+
+```xml title=AndroidManifest.xml
+  <uses-permission android:name="android.permission.RECORD_AUDIO"/>
+```
+
+On iOS you must set `NSCameraUsageDescription` in `Info.plist` file. You can
+edit this file in Xcode. This value is a description that is shown when iOS asks user
+for camera permission.
+
+```xml title=ios/jellyfish-dashboard/Info.plist
+<key>NSCameraUsageDescription</key>
+<string> 🙏 🎥 </string>
+```
+
+Similarly, for audio there is `NSMicrophoneUsageDescription`.
+
+```xml title=ios/jellyfish-dashboard/Info.plist
+<key>NSMicrophoneUsageDescription</key>
+<string> 🙏 🎤 </string>
+```
+
+For screencast there is more configuration needed, it's described [here](https://github.com/jellyfish-dev/react-native-client-sdk#ios).
+
+We also suggest setting background mode to `audio` so that the app doesn't
+disconnect when it's in the background:
+
+```xml title=ios/jellyfish-dashboard/Info.plist
+<key>UIBackgroundModes</key>
+<array>
+  <string>audio</string>
+</array>
+```
+
+  </TabItem>
+   <TabItem value="expo-bare" label="Expo Bare workflow"> 
+   
+   You have two options here. You can follow configuration instructions for
+React Native (Expo Bare workflow is a React Native project after all) or if
+you're using `expo prebuild` command to set up native code you can add our Expo
+plugin. Just add it to `app.json`:
+
+```json title:app.json
+{
+  "expo": {
+    "name": "example",
+    //...
+    "plugins": ["@jellyfish-dev/react-native-membrane-webrtc"]
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+## Add components library
+
+For your convenience, we've prepared a library with nice-looking components
+useful for following this tutorial. Feel free to use standard React Native components or your own components though!
+
+```bash
+npx expo install @expo/vector-icons expo-barcode-scanner expo-font @expo-google-fonts/noto-sans @jellyfish-dev/react-native-jellyfish-components @react-navigation/native-stack
+```
+
+**_You'll also need to install [Reanimated library (3.3.0)](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started) and [React Navigation (6.1.7)](https://reactnavigation.org/)_**
+
+:::tip
+Run `pod install` in the /ios directory to install the new pods
+:::
+
+## Connecting to the server and joining the room
+
+Our app will consist of two screens. The first one allows the user to type, paste or
+scan the peer token and connect to the room. The second screen shows room
+participants with their video tracks.
+
+### Screens
+
+For managing screens I've used React Navigation library, you're free though to
+use another one. Our app will contain two screens: Connect screen and Room
+screen:
```

Comment:
Keep using `we` so

```suggestion
For managing screens we will use React Navigation library, but feel free to pick whatever suits you. 
```

We already mentioned screens in the previous section so I would put their names there. See previous comment